fondutils/__version__.py

```diff
@@ -9,16 +9,4 @@
 # license that can be found in the LICENSE file or at
 # https://opensource.org/licenses/MIT.
 #
-
-"""Package metadata declaration."""
-
-__title__ = "fond-utils"
-__description__ = "FOND Utilities"
-__url__ = "https://github.com/AI-Planning/fond-utils/"
 __version__ = "0.1.2"
```

Comment:
So I guess a new tag will trigger a commit that updates this? I thought there would be some need for github action magic to make this happen, no?

Reply:
mmmm don't know exactly how it works, but when I did `pip  install .`, it re-generated a full `__version__.py` that says not to track it in git:


![image](https://github.com/user-attachments/assets/6953c090-ccb1-49c9-8255-fef543d55da7)

Reply:
I drop file `__version__.py` from git tracking...

see how the above produced the new version. No idea how though.. :smile:

Reply:
OK no need to have any version file at all; that's a bit old and is only done for backwards compatibility. Modern style is all dynamic via the toml file and `setuptools-scm` package

See https://github.com/AI-Planning/fond-utils/issues/20#issuecomment-2834253347

Reply:
I am now happy with what it is, simple and robust, all versioning is dynamic, no need to ever do any boost of version except when we decide to release a stable version.